site/content/contribute/more-info/mobile/build-your-own/_index.md

```diff
@@ -14,10 +14,13 @@ At Mattermost, we build and deploy the Apps using a CI pipeline. The pipeline ha
 
 As an alternative we've also created a set of **scripts** to help automate build tasks. Learn more about the scripts by reviewing the {{< newtabref href="https://github.com/mattermost/mattermost-mobile/blob/master/package.json" title="package.json" >}} file.
 
+---
 {{<note "Note:">}}
 **Note:** By using the **scripts**, {{< newtabref href="https://docs.fastlane.tools/#choose-your-installation-method" title="Fastlane" >}} and other dependencies will be installed in your system.
 {{<note "Note:">}}
```

Comment:
```suggestion
{{</note>}}
```
Please make sure to close the `note` tag at the end of the note. :slightly_smiling_face: